core/src/test/java/org/infinispan/test/AbstractInfinispanTest.java

```diff
@@ -185,6 +186,51 @@ protected void killSpawnedThreads() {
       }
    }
 
+   /**
```

Comment:
wouldn't be more useful to have this login in commons-tests? 
probably in `CommonsTestingUtil`... or create a `BlockHoundUtil` class?
wdyt?

Reply:
Sure, I can add it to BlockHoundHelper and make the class public.